### DISCUSSION TITLE
Fullscreen dialogs

### DISCUSSION THREAD

**Discussion by snoopy** [orbitalquark/textadept#575] [Q&A] [answered]
Is there a way to make dialogs cover the entire screen by default?
I noticed you can resize them manually by dragging but there seems to be no way to make them bigger by default. 
`ui.size` apparently only includes the main window size.

**Comment by orbitalquark** [comment, marked as answer]
No, this is not possible. Is there a dialog in particular that appears too small? Usually Qt does a good job resizing its dialogs to their contents.

**Reply by snoopy** [reply]
No, I was just wondering if this is possible in general.
